lib/DxilDia/DxilDiaSymbolManager.cpp

```diff
@@ -1070,7 +1070,7 @@ STDMETHODIMP dxil_dia::hlsl_symbols::LocalVariableSymbol::get_sizeInUdt(
     return E_INVALIDARG;
   }
 
-  static constexpr DWORD kBitsPerByte = 8;
+  //static constexpr DWORD kBitsPerByte = 8;
```

Comment:
If unused, I prefer to just remove it entirely

Reply:
Fixed.